typescript/src/bitcoin.ts

```diff
@@ -359,3 +359,62 @@ export function createKeyRing(
 export function computeHash160(text: string): string {
   return hash160.digest(Buffer.from(text, "hex")).toString("hex")
 }
+
+/**
+ * Encodes a public key hash into a P2PKH/P2WPKH address.
+ * @param publicKeyHash - public key hash that will be encoded. Must be an
+ *        unprefixed hex string (without 0x prefix).
+ * @param witness - If true, a witness public key hash will be encoded and
+ *        P2WPKH address will be returned. Returns P2PKH address otherwise
+ * @param network - Network that the address should be encoded for.
+ *        For example, `main` or `testnet`.
+ * @returns P2PKH or P2WPKH address encoded from the given public key hash
+ */
+export function encodeToBitcoinAddress(
+  publicKeyHash: string,
+  witness: boolean,
+  network: string
+): string {
+  return witness
+    ? bcoin.Address.fromWitnessPubkeyhash(
+        Buffer.from(publicKeyHash, "hex")
+      ).toString(network)
+    : bcoin.Address.fromPubkeyhash(Buffer.from(publicKeyHash, "hex")).toString(
+        network
+      )
```

Comment:
Let's make it a little bit cleaner:
```suggestion
  const buffer = Buffer.from(publicKeyHash, "hex")
  const address = witness
    ? bcoin.Address.fromWitnessPubkeyhash(buffer)
    : bcoin.Address.fromPubkeyhash(buffer)
  return address.toString(network)
```

Reply:
Fixed in f52b614cf9ce40c95f10574c4a1d501765880180